docs/concepts/policy/pod-security-policy.md

```diff
@@ -354,6 +354,15 @@ several security mechanisms.
 
 ## Policy Reference
 
+### Privileged
+
+**Privileged** - determines if any container in a pod can enable privileged mode.
+By default a container is not allowed to access any devices on the host, but a 
+"privileged" container is given access to all devices on the host. This allows
+the container nearly all the same access as processes running on the host.
+This is useful for containers that want to use linux capabilities like
```

Comment:
This may confusing a reader because for managing linux capabilities we have dedicated fields (`allowedCapabilities`, `defaultAddCapabilities`, and `requiredDropCapabilities`).